src/Microsoft.TestPlatform.Common/Utilities/FakesUtilities.cs

```diff
@@ -34,9 +35,9 @@ public static class FakesUtilities
     /// <param name="sources">test sources</param>
     /// <param name="runSettingsXml">runsettings</param>
     /// <returns>updated runsettings for fakes</returns>
-    public static string GenerateFakesSettingsForRunConfiguration(string[] sources, string runSettingsXml)
+    public static string GenerateFakesSettingsForRunConfiguration(IDictionary<string, Framework> sourceToFrameworkMap, string runSettingsXml)
```

Comment:
What about simply `sourceToFramework`? I am not a big fan of "map" suffix in C# (feels Java). If you really need the suffix, I would vote for `Dictionary`.

Reply:
too late, I already used in in many other places, and honestly, reading sourceToFramework, does not give me the same understanding as reading sourceToFrameworkMap, or sourceToFrameworkDictionary

Reply:
Not too late to not introduce more `map` instead of `dictionary`, PR isn't yet merged :)